packages/app/components/common/ErrorBoundary.tsx

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
```

Comment:
Could we remove this and add an underscore to the variables, now that the linter rules have been updated?

Reply:
this is more I'm scared/don't quite know if `componentDidCatch` will play nice if we rename its arguments